1.2/index.html

```diff
@@ -1525,19 +1525,13 @@ <h2 class="no-num no-toc no-ref" id="contents">Table of Contents</h2>
     <li>
      <a href="#inline-representation"><span class="secno">6</span> <span class="content">Inline Representations</span></a>
      <ol class="toc">
-      <li>
-       <a href="#inline-classes"><span class="secno">6.1</span> <span class="content">Classes for Inline Representation</span></a>
-       <ol class="toc">
-        <li><a href="#ocr_glyph"><span class="secno">6.1.1</span> <span class="content"><span>ocr_glyph</span></span></a>
-        <li><a href="#ocr_glyphs"><span class="secno">6.1.2</span> <span class="content"><span>ocr_glyphs</span></span></a>
-        <li><a href="#ocr_dropcap"><span class="secno">6.1.3</span> <span class="content"><span>ocr_dropcap</span></span></a>
-        <li><a href="#ocr_chem"><span class="secno">6.1.4</span> <span class="content"><span>ocr_chem</span></span></a>
-        <li><a href="#ocr_math"><span class="secno">6.1.5</span> <span class="content"><span>ocr_math</span></span></a>
-        <li><a href="#superscript-and-subscript"><span class="secno">6.1.6</span> <span class="content">Superscript and Subscript</span></a>
-        <li><a href="#non-breaking-space"><span class="secno">6.1.7</span> <span class="content">Non-breaking space</span></a>
-        <li><a href="#non-default-spaces"><span class="secno">6.1.8</span> <span class="content">Non-default spaces</span></a>
-        <li><a href="#hyphenation"><span class="secno">6.1.9</span> <span class="content">Hyphenation</span></a>
-       </ol>
+      <li><a href="#unrecognized"><span class="secno">6.1</span> <span class="content">Unrecognized characters and words</span></a>
+      <li><a href="#dropcap"><span class="secno">6.2</span> <span class="content">Dropcap</span></a>
+      <li><a href="#formulas"><span class="secno">6.3</span> <span class="content">Mathematical and chemical formulas</span></a>
+      <li><a href="#sub-sup"><span class="secno">6.4</span> <span class="content">Superscript and Subscript</span></a>
+      <li><a href="#whitespace"><span class="secno">6.5</span> <span class="content">Whitespace</span></a>
+      <li><a href="#hyphenation"><span class="secno">6.6</span> <span class="content">Hyphenation</span></a>
+      <li><a href="#ruby"><span class="secno">6.7</span> <span class="content">Ruby characters</span></a>
```

Comment:
I've moved the paragraph there before and probably will again once I get to the fonts/language section :)